index.js

```diff
@@ -12,11 +12,11 @@ module.exports = class HyperBundle extends EventEmitter {
       opts = key
       key = null
     }
-    const { _checkout, _db, _files, onwait } = opts
-    this._onwait = onwait || null
+    const { _checkout, _db, _files } = opts
+    this._opts = opts
```

Comment:
can you name this `this._options` instaead

Reply:
Done and I added a separate commit to delete `keyPair` from the options. Hopefully, it makes sense.